package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "mocha": "*",
-    "radar_client": "*",
+    "radar_client": "git://github.com/zendesk/radar_client.git#pobrien/refactor_init_sync_get",
```

Comment:
yep, I got the tests to pass locally for radar_client, and then did not commit.  But it has passed now.